dev.rollup.config.js

```diff
@@ -40,20 +43,21 @@ export default Object.assign({}, umdConfig, {
         dir: tmpDestPath
     }),
     watch: {
-        clearScreen: false
+        clearScreen: false,
```

Comment:
we need to add prettier to this repo.  i usually omit trailing commas, but, that's completely a style thing and solved by prettier.

no need to change this back.  just making an observation

Reply:
I also prefer to omit them, I guess I forgot that one during development ;)